go/vt/vtgate/planbuilder/operators/queryprojection.go

```diff
@@ -95,17 +97,12 @@ func (aggr Aggr) NeedsWeightString(ctx *plancontext.PlanningContext) bool {
 	return aggr.OpCode.NeedsComparableValues() && ctx.SemTable.NeedsWeightString(aggr.Func.GetArg())
 }
 
-func (aggr Aggr) GetTypeCollation(ctx *plancontext.PlanningContext) evalengine.Type {
+func (aggr Aggr) GetParameterType(ctx *plancontext.PlanningContext) evalengine.Type {
 	if aggr.Func == nil {
-		return evalengine.Type{}
+		return evalengine.NewType(sqltypes.Unknown, collations.Unknown)
```

Comment:
@systay What do you think of a new `evalengine.NewUnknownType()` function to replace this one and all other cases like this with?

Reply:
Sure, makes sense I think